ietf-document/draft-0rtt-bdp/draft-kuhn-quic-0rtt-bdp-07.txt

```diff
@@ -49,144 +70,198 @@ Copyright Notice
+   established during the handshake are used.
 
-   This memo proposes the sharing of additional transport parameters to
-   provide a faster startup of flows and a better buffer management for
-   the data that the client receives, i.e. the ingress traffic.  To do
-   so, an extension, 'BDP metadata', is proposed.  Candidate transport
-   parameters are discussed in Section 3.
+   This document proposes an extension to the transport parameters that
+   are shared during the 0-RTT phase to allow resumption using transport
+   and connection properties that were discovered in previous
+   connections.  These additional parameters aim to provide faster
+   startup of flows and better buffer management.  The BDP metadata
+   extension is proposed and candidate parameters are discussed in
+   Section 3.
 
 2.  Motivation
 
-   Egress acceleration like 0-RTT "initial_max_data" alone does not
-   improve the time-to-service.  While "initial_max_data" improves the
-   egress time to server, the BDP metadata extension aims at improving
-   ingress traffic delivery.
+   Reducing the number of round-trips required to start a connection is
+   an important way to reduce setup time and lower overall connection
+   latency. 0-RTT mechanisms that allow a client to get feed requests to
+   a server in the first RTT do not alone improve the total time-to-
+   service.  The BDP extension described in this document aims to
+   improve traffic delivery by allowing the connection to short cut slow
+   RTT based processes that grow connection parameters.
 
    Currently each side has its proprietary solution to measure and to
    store path characteristics.  Recalling the parameters of a previous
-   connection would:
+   connection would allow:
 
-   o  allow server and client to immediately recover parameters that do
-      not need to be negociated or estimated again (minRTT, MTU,
-      bottleneck capacity, etc.)
 
-   o  allow the client to prepare the right resources
 
 
-
-Kuhn, et al.             Expires October 1, 2020                [Page 2]
+Kuhn, et al.            Expires November 9, 2020                [Page 3]
 
-Internet-Draft             Transport for 0-RTT                March 2020
+Internet-Draft             Transport for 0-RTT                  May 2020
+
+
+   o  Server and client can re-initialise the state from previously
+      established parameters (i.e., minRTT, MTU, bottleneck capacity,
+      etc.)
 
+   o  The client to prepare the right resources
 
-   o  allow server side adaptation to non-default path characteristics
+   o  Server side adaptation to non-default path characteristics
+
+   There is an interest in sharing transport information across multiple
+   connections.  Indeed, our proposal is inline with the motivation of
+   [I-D.ietf-tcpm-2140bis].  While our proposal aims at improving 0-RTT
+   mechanisms, [I-D.ietf-tcpm-2140bis] considers the sharing of
+   transport parameters between connections sharing the same host.
```

Comment:
/sharing the same host/originating from the same host/ ... or something else?
... To me, I-D.ietf-tcpm-2140bis is about a sender storing path information. Whereas this ID is about a client storing the information and later transmission to the server at the start of a connection. This has the advantage of not requiring the server to retain additional state for each client.
... is that so?

Reply:
Ok i think i better see what you meant by pointing to 2140bis. I have proposed some text in the commit "improve ref to 2140"